GVFS/GVFS.Common/GVFSConstants.cs

```diff
@@ -33,7 +33,7 @@ public static class GitConfig
             public const string HooksPrefix = GitConfig.GVFSPrefix + "clone.default-";
             public const string GVFSTelemetryId = GitConfig.GVFSPrefix + "telemetry-id";
             public const string HooksExtension = ".hooks";
-            public const string UpgradeRing = GVFSPrefix + "upgrade-ring";
+            public const string UpgradeRing = "upgrade-ring";

```

Comment:
If we're lifting into our own config file, upgrade.ring so we could have other options in the upgrade section?

Reply:
Sure. Renamed it.